exercises/collatz-conjecture/collatz-conjecture.spec.js

```diff
@@ -0,0 +1,31 @@
+import steps from './collatz-conjecture';
```

Comment:
It seems it follows [canonical data](https://github.com/exercism/problem-specifications/blob/master/exercises/collatz-conjecture/canonical-data.json).